src/battle/area_omo/512E50.c

```diff
@@ -8,6 +8,21 @@ INCLUDE_ASM(s32, "battle/area_omo/512E50", func_80218880_512E50);
 
 INCLUDE_ASM(s32, "battle/area_omo/512E50", func_80218B38_513108);
 
-INCLUDE_ASM(s32, "battle/area_omo/512E50", func_80218CF4_5132C4);
+/// checks if all actions are available.
+/// used by Spy Guys.
+ApiStatus func_80218CF4_5132C4(ScriptInstance* script, s32 isInitialCall) {
+    BattleStatus* status = &gBattleStatus;
+    Bytecode* args = script->ptrReadPos;
+
+    if ((status->hammerLossTurns == -1) || 
+        (status->jumpLossTurns == -1) || 
+        (status->itemLossTurns == -1)) {
```

Comment:
Sorry, I missed this earlier. You don't need the extra parens around each == condition - they're only required for `&` (AND) flag check conditions because C has stupid precedence rules.
```suggestion
    if (status->hammerLossTurns == -1 || 
        status->jumpLossTurns == -1 || 
        status->itemLossTurns == -1
    ) {
```